docs/components/VisuallyHidden.tsx

```diff
@@ -0,0 +1,17 @@
+import { css } from '@emotion/react';
+import React from 'react';
+
+const VISUALLY_HIDDEN_STYLES = css`
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+`;
+
+export const VisuallyHidden = (props: React.HTMLAttributes<HTMLElement>) => (
```

Comment:
ℹ️ This component allows information to be invisible visually BUT still being announced by screen readers. This is pretty convenient for forms where don't want to have a label for example (like the theme select element) while keeping a full meaning for people that don't see the screen.